code/modules/clothing/under/accessories/armor.dm

```diff
@@ -1,11 +1,25 @@
+/*
+// This file holds all of the accessories used as part of the modular armor system. At some point it might be wise to split this into multiple files.
+*/
+
+/obj/item/clothing/accessory/armor
```

Comment:
How would this play with the chameleon kit? Just checking.

Reply:
It doesn't have any effect, as far as I'm aware?